src/lib/viewers/iframe/IFrameLoader.js

```diff
@@ -30,9 +36,9 @@ class IFrameLoader extends AssetLoader {
     determineViewer(file, disabledViewers = [], viewerOptions = {}) {
         const isDicomFile = file.extension === 'boxdicom';
         const disableDicom = getProp(viewerOptions, 'IFrame.disableDicom');
-        // The IFrame viewer is disabled when the file is a Boxdicom file and the disableDicom viewer option is enabled
+        // Removes boxdicom as a supported extension when the file is a Boxdicom file and the disableDicom viewer option is enabled
```

Comment:
Do we even need to check if `isDicomFile` is true anymore? If `disableDicom` option is passed in, would it work if we just remove it from the `this.viewers`?

Reply:
I think you're right, it doesn't really matter about the current file, we should just remove the dicom viewer from the supported viewers if the disableDicom is true.

Reply:
May want to update the comment as well to be consistent with the code